### PR TITLE
Hilbert Fixes & Tweaks

### LINKED ISSUE #68072: Hilbert's Hotel ruin Tram caller button doesn't work
## Round ID:
don't remember

## Testmerges:

don't remember

## Reproduction:

> go to hilbert's hotel
realize that the tram spawns in the middle of the ruin
attemot to call the tram
button says tram isn't working
walk over to tram
it works just fine

maybe caused by #66657?

### PR DESCRIPTION
**Pull request by Fikou**
## About The Pull Request
Hilbert Research Facility tram buttons work now.
Hilbert Research Facility's tram no longer has engines.
Redoes the overlay effect on the broken tv in hilbert lore room by making it its own type.
Makes the tele porta turrets their own type.
Adds support for html in lore terminals, and makes Roman's emails use them.
fixes https://github.com/tgstation/tgstation/issues/68072

## Why It's Good For The Game
some neat stuff that hilbert needed

## Changelog
:cl:
fix: Hilbert Research Facility tram buttons work now.
qol: Adds support for html in lore terminals, and makes Hilbert Research Facility emails use them.
/:cl:

### CODE REVIEW

**Comment by private-tristan** [comment]
fixes #68072

File: code/game/machinery/buttons.dm [hunk 1]
```diff
@@ -324,11 +324,13 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/button/door, 24)
 	device_type = /obj/item/assembly/control/tram
 	req_access = list()
 	id = 1
+	var/lift_id = MAIN_STATION_TRAM
```
Comment (Mothblocks) [inline code/game/machinery/buttons.dm:327]: Document this since we're intending to promote it to mappers

**Comment by Mothblocks** [comment]
I wish it was markdown

File: code/game/machinery/buttons.dm (hunk 1 above)
Comment (Fikou) [inline code/game/machinery/buttons.dm:327]:
```suggestion
	/// The specific lift id of the tram we're calling.
	var/lift_id = MAIN_STATION_TRAM
```

**Pull request merged**